internal/service/constants.go

```diff
@@ -0,0 +1,13 @@
+package service
+
+const (
+	TestNamespace               = "orantest"
+	AlarmConfigMapName          = "oran-alarms-sub"
+	InfraInventoryConfigMapName = "oran-infra-inventory-sub"
+	FieldOwner                  = "oran-o2ims"
+)
+
+const (
+	SUBSCRPTION_TYPE_ALARM                     = "ALARM"
```

Comment:
Should these be `SubscriptionTypeAlarm` and `SubscriptionTypeInfrastructureInventory` to be consistent with the typical naming for constants in Go?